packages/apollo-language-server/src/project/base.ts

```diff
@@ -106,6 +108,8 @@ export abstract class GraphQLProject implements GraphQLSchemaProvider {
 
   protected abstract initialize(): Promise<void>[];
 
+  abstract getProjectStats(): any;
```

Comment:
Does it make sense to enforce a type here? Or only at the concrete implementation? Here makes sense to me but you've also got some more context than I do.